prac_06/guitars.py

```diff
@@ -0,0 +1,37 @@
+"""
+CP1404 Practical 06 - Guitar program using class and methods
+Time estimated: 50 minutes
+Time taken: 40 minutes
+"""
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Program to store and display details of a user's guitars."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
```

Comment:
You maybe did not need to make a guitar_to_add object here, and could of done  
guitars.append(Guitar(name, year, cost))